### DISCUSSION TITLE
Is the workflow runs array chronologically sorted?

### DISCUSSION THREAD

**Discussion by weibeld** [community/community#24372] [Apps, API and Webhooks]
<p>The <a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-workflow-runs" rel="noopener nofollow ugc">List workflow runs</a> API endpoint returns an array of workflow runs for a specific workflow. Are these workflow runs guaranteed to be sorted according to the order in which they were executed (i.e. based on the <code>run_number</code> field wit the newest workflow run first)?</p>
<p>In example request, the workflow runs are indeed sorted (newest first), but I wonder if this is a guaranteed behaviour.</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/weibeld">@weibeld</a> :wave:</p>
<p>Returns from this endpoint should be sorted by the <code>created_at</code> value, not necessarily <code>run_number</code>.</p>
<p>Unsure if that simple answer is enough to suit your needs, but if you have feedback, please consider submitting through this form:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">support.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">Share feedback - GitHub Support</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>…where it will be reviewed directly by our PMs for consideration.</p>
<p>If you have any followup questions, please let us know and we’ll do our best! :bow:</p>

**Reply by AlessandroVol23** [reply]
Is it somehow possible to change the sort order in the API?
